sdks/swift/Mentat/Mentat/Core/TypedValue.swift

```diff
@@ -0,0 +1,75 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Mentatlib
```

Comment:
Can we get a better name for this? It's not clear why we have two "Mentat lib"s.

Perhaps `libmentat` for the native code?